source/index.html

```diff
@@ -130,10 +130,213 @@ <h1 class="row-title">Top Breakfast Recipes</h1>
+                                        <option value="">None selected</option>
+                                        <option value="descending">Descending</option>
+                                        <option value="ascending">Ascending</option>
+                                    </select>
+                                </div>
+                            </div>
+                            <hr class="solid">
+                            <div class="filters-meal-type">
+                                <div class="filters-meal-type">
+                                <div class="filter-meal-type-input">
+                                    <label class="sort-filter-label">Meal Type:</label><br>
+                                    <select name="meal-type" id="meal-type" value="">
+                                        <option value="">None selected</option>
+                                        <option value="main course">Main course</option>
+                                        <option value="side dish">Side dish</option>
+                                        <option value="dessert">Dessert</option>
+                                        <option value="appetizer">Appetizer</option>
+                                        <option value="salad">Salad</option>
+                                        <option value="bread">Bread</option>
+                                        <option value="breakfast">Breakfast</option>
+                                        <option value="soup">Soup</option>
+                                        <option value="beverage">Beverage</option>
+                                        <option value="sauce">Sauce</option>
+                                        <option value="marinade">Marinade</option>
+                                        <option value="fingerfood">Fingerfood</option>
+                                        <option value="snack">Snack</option>
+                                        <option value="drink">Drink</option>
+                                    </select>
+                                </div>
+                                </div>
+                            </div>
+                            <hr class="solid">
+                            <div class="filters-cuisine">
+                            <div class="filters-cuisine-header">
+                                <label class="sort-filter-label">Filter by cuisine:</label>
+                                <button class="cuisine-collapse" id="cuisine-collapse" type="button">↑</button>
+                            </div>
+                            <div class="filters-cuisine-body" id="cuisine-collapse-body">
+                                <div class="filter-cuisine-input">
+                                <input type="checkbox" id="cuisine1" name="cuisine1" value="African">
+                                <label for="cuisine1">African</label><br>
```

Comment:
Why are the cuisines named cuisine# rather than just what they are (i.e. instead of name = cuisine1, name = African)? If we don't expect to have to reference these I might approv.

Reply:
I think I based it off of the code I was using as a reference. I don't think they're used in any way outside of being there.